### PR TITLE
[zh] Sync #15004 move HBONE section into its own doc into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync #15004 move HBONE section into its own doc into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/ambient/architecture/tls-tunnel/index.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+- [HTTP CONNECT](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/CONNECT)
+- [双向 TLS（mTLS）](https://datatracker.ietf.org/doc/html/rfc8446)
+
+HTTP CONNECT 用于建立隧道连接，mTLS 用于保护和加密该连接，
+HTTP/2 用于在单个安全和加密隧道上复用应用程序连接流，并传送其他流级元数据。
+
+## 安全和租户 {#security-and-tenancy}
+
+根据 mTLS 规范的强制要求，每个底层隧道连接必须具有唯一的来源身份和唯一的目标身份，
+并且必须使用这些身份为该连接建立加密。
```
Comment (windsonsea) [inline content/zh/docs/ambient/architecture/tls-tunnel/index.md:26]:
```suggestion
并且必须使用这些身份为该连接进行加密。
```

File: content/zh/docs/ambient/architecture/tls-tunnel/index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+这意味着通过 HBONE 协议到相同目标身份的应用程序连接将在相同的共享、
```
Comment (windsonsea) [inline content/zh/docs/ambient/architecture/tls-tunnel/index.md:28]:
```suggestion
这意味着通过 HBONE 协议从应用程序到同一个目标身份的连接将在同一个共享的、
```

File: content/zh/docs/ambient/architecture/tls-tunnel/index.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+加密和安全的底层 HTTP/2 连接上进行多路复用 - 实际上，
```
Comment (windsonsea) [inline content/zh/docs/ambient/architecture/tls-tunnel/index.md:29]:
```suggestion
加密的和安全的底层 HTTP/2 连接上进行多路复用。实际上，
```

File: content/zh/docs/ambient/architecture/tls-tunnel/index.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+每个唯一的来源和目标都必须获得自己专用的安全隧道连接，
```
Comment (windsonsea) [inline content/zh/docs/ambient/architecture/tls-tunnel/index.md:30]:
```suggestion
即使该底层专用连接正在处理多个应用程序级连接，
```

File: content/zh/docs/ambient/architecture/tls-tunnel/index.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+甚至如果该底层专用连接正在处理多个应用程序级连接。
```
Comment (windsonsea) [inline content/zh/docs/ambient/architecture/tls-tunnel/index.md:31]:
```suggestion
每个唯一的来源和目标也必须获取自己专用的安全隧道连接。
```

**Review by windsonsea — COMMENTED** [review]
for fluent.

**Comment by windsonsea** [comment]
Need to rebase and resolve the conflicts.

**Comment by wilsonwu** [comment]
> Need to rebase and resolve the conflicts.

Yes, related docs have lots of changes, I will resolve all after other PRs merged.

**Review by windsonsea — APPROVED** [review]
lgtm

**Comment by wilsonwu** [comment]
Close this PR by the conflicts problem.

**Pull request closed without merging**